src/chainstate/coordinator/tests.rs

```diff
@@ -960,6 +969,229 @@ fn test_sortition_with_reward_set() {
     }
 }
 
+#[test]
+fn test_sortition_with_burner_reward_set() {
```

Comment:
Can this test be updated so there's an odd number of non-burn addresses in the recipient set?  Then, we can see that it works for the "last" recipient (whose block-commit will include a burn address).

Reply:
Sure -- updated in 00ec45dc0